src/game.cpp

```diff
@@ -12821,11 +12821,11 @@ void game::vertical_notes( int z_before, int z_after )
             if( z_after > z_before && ter->has_flag(known_up) &&
                 !ter2->has_flag(known_down) ) {
                 overmap_buffer.set_seen(cursx, cursy, z_after, true);
-                overmap_buffer.add_note(cursx, cursy, z_after, _(">:W;AUTO: goes down"));
+                overmap_buffer.add_note(cursx, cursy, z_after, (std::string)">:W;" + _("AUTO: goes down"));
```

Comment:
This seems to be a good place for `string_format()`.

Reply:
What's so bad about string concatenation? (Beside the C-style cast here.)